src/client/src/components/SearchBar.js

```diff
@@ -0,0 +1,34 @@
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class SearchBar extends Component {
+
+    state = {
+        stockName: ''
+    }
+
+    searchFor = () => {
```

Comment:
Why is findOpportunities passed into SearchBar instead of being part of SearchBar?

Reply:
If it's to make it more general, rename findOpportunities here to something else.

Reply:
I need to lift state up, cannot set state in SearchBar.

Reply:
changed name